clang/test/Driver/DTLTO/dtlto.c

```diff
@@ -0,0 +1,44 @@
+/// Check DTLTO options are forwarded to the linker.
+
+// REQUIRES: lld
+
+// RUN: echo "-target x86_64-linux-gnu \
```

Comment:
`-target ` has been deprecated for a long time. Use `--target=` for new tests

Reply:
Done. Thanks. Apologies, I now recall you writing about this previously.